ArrayExtensions is not defined as a dependency

```Packages\com.varneon.vudon.quick-menu\Runtime\Udon Programs\QuickMenu.cs(6,21): error CS0234: The type or namespace name 'ArrayExtensions' does not exist in the namespace 'Varneon.VUdon' (are you missing an assembly reference?)```

--- a/Packages/com.varneon.vudon.quick-menu/package.json
+++ b/Packages/com.varneon.vudon.quick-menu/package.json
@@ -1,7 +1,7 @@
 {
   "name": "com.varneon.vudon.quick-menu",
   "displayName": "VUdon - Quick Menu",
-  "version": "0.1.0-alpha.8",
+  "version": "0.1.0-alpha.9",
   "unity": "2019.4",
   "unityRelease": "29f1",
   "description": "HUD quick menu prefab for VRChat worlds",

--- a/Packages/com.varneon.vudon.quick-menu/package.json
+++ b/Packages/com.varneon.vudon.quick-menu/package.json
@@ -12,10 +12,12 @@
   "resolutionStrategy": "highestMinor",
   "dependencies": {
     "com.vrchat.udonsharp": "1.1.7",
+    "com.varneon.vudon.array-extensions": "0.3.0",
     "com.varneon.vudon.menus": "0.1.0-alpha.4"
   },
   "vpmDependencies": {
     "com.vrchat.udonsharp": ">=1.1.7 <2.0.0",
+    "com.varneon.vudon.array-extensions": ">=0.3.0",
     "com.varneon.vudon.menus": ">=0.1.0-alpha.3 <1.0.0"
   }
 }

